### PR TITLE
Myst rst2myst conversion process

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

**Comment by stevepiercy** [comment]
I think this is pretty solid. I'd suggest making a clean clone to avoid destroying your working clone, just in case I goofed.

Please run through it to see if I missed anything.

There are two TODOs that need more thought at the end, specifically how to move the .md files back to the parent directory, and removing the .rst files (except `README.rst` and `CHANGES.rst`).

Feedback greatly appreciated.

**Review by pbauer — APPROVED** [review]
Solid work

**Comment by stevepiercy** [comment]
I made several mistakes and there were a few omissions. It's a good thing I walked through this again. _Now_ it's ready for final review.

I'd like another pair of eyes on this before I merge it.

I will do two PRs:

- Fix syntax errors that cause build warnings in .rst or .md.
- Run through the conversion process, after this PR is merged.

**Pull request merged**
